packages/fxa-payments-server/src/routes/Checkout/en.ftl

```diff
@@ -1,7 +1,6 @@
 ## Routes - Checkout - New user
 
 # TODO - Remove once branding sticks
-new-user-step-1 = 1. Create a { -brand-name-firefox } account
```

Comment:
We have other follow ups to remove these 'legacy' strings, so maybe don't remove here? If you want to remove it anyway, please remove the TODO comment above.

Reply:
Thanks for the catch! I was updating from my previous commit but tried to leave these strings because of the comments. Don't want to mess with the flow. I'll revert. 👍